src/Core/src/Platform/iOS/ButtonExtensions.cs

```diff
@@ -83,5 +84,27 @@ public static void UpdatePadding(this UIButton platformButton, Thickness padding
 #pragma warning restore CA1422 // Validate platform compatibility
 #pragma warning restore CA1416
 		}
+
+		internal static void UpdateAttributedTitle(this UIButton platformButton, IFontManager fontManager, ITextStyle textStyle)
+		{
+			// Any text update requires that we update any attributed string formatting
+			var uiFontAttribute = fontManager.GetFont(textStyle.Font, UIFont.ButtonFontSize);
+			var attributedString = new NSMutableAttributedString(new NSAttributedString(platformButton.CurrentTitle!));
```

Comment:
Looks like there may some situations where this platformButton.CurrentTitle! is null and crashes. Perhaps a null check and bailing could be useful here if the title is not populated yet?

https://gist.github.com/tj-devel709/10346a24bbb84b04a1f8dc4e15777410

Reply:
You're right! I've added a null check